app/src/main/java/com/belatrixsf/allstars/ui/common/LoadMoreBaseAdapter.java

```diff
@@ -13,7 +13,7 @@
 /**
  * Created by PedroCarrillo on 4/26/16.
  */
-public abstract class LoadMoreBaseAdapter<T,VH extends RecyclerView.ViewHolder> extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+public abstract class LoadMoreBaseAdapter<T> extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
first it was intented when the recycler view had only one type of view holder, but if we consider other view types we can't set the ViewHolder as an specific class of ViewHolder and we just use RecyclerView.ViewHolder